src/Network/Network/ExpressRouteCircuit/Peering/RemoveAzureExpressRouteCircuitPeeringConfigCommand.cs

```diff
@@ -100,21 +100,24 @@ public override void Execute()
 
                 else if (peering.PeeringType == MNM.ExpressRoutePeeringType.AzurePrivatePeering)
                 {
-                    if (AddressTypeUtils.IsIpv4(this.PeerAddressType) || string.IsNullOrWhiteSpace(this.PeerAddressType))
```

Comment:
the null check should be before the Ipv4 check.

Reply:
sure, i will do that.